uwrt_mapviz/CMakeLists.txt

```diff
@@ -0,0 +1,206 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(uwrt_mapviz)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  mapviz
```

Comment:
These aren't needed for build. the only things in this file for "launch/config only" packages should be install directives for the config and launch folders. Here is an example:
https://github.com/ros-simulation/gazebo_ros_demos/tree/kinetic-devel/rrbot_control